### ISSUE TITLE
Scroll-Function in Article Templates

### ISSUE THREAD

**Issue by bluesidra** [worldanvil/worldanvil-bug-tracker#1272]
**World Anvil Username**:bluesidra7

**World / Campaign / RPG Character / Manuscript Title**:stardust

**Feature**: Article Templates

**Related URLS** (To public & visible pages add screenshots otherwise):

**Describe the Issue**
- when scrolling over the text-fields in the "Sections"-section of any article template, the scroll function behaves different from the "Content"-section
- when having the mouse over a text-field, the scroll-function (mousewheel etc) doesn't scroll the page, even if the text-field is empty and has no scroll-bar on its own. You have to move the mouse out of the field to scroll further down the page.
- Note that this only happens if the mouse comes to rest over the field. If you start scrolling down on eg. the article title, the scrolling won't stop as soon as you mouseover a text-field. But if you stop, the scrolling function doesn't work any longer. 
- Contrast to scroll-behaviour on the "Content"-section: If the cursor comes to rest on a text-field and you start scrolling, it will first scroll through the textfield, and, once it reaches the end, scroll the page. If there is no scroll-bar in the text-field, it will immediately scroll the page. 

**To Reproduce**
1. Open any article template (I've encountered it in all templates so far, except for the Prose template)
2. Go to the "Section"-section
3. Scroll down until the mouse comes to rest on an empty text-field
4. Scroll down again
5. Page doesn't move

**Expected behavior**
I would expect the same behaviour as on the "Content" page, where the page will be scrolled once the end of the text-field's scroll-bar is reached. 

**Screenshots**
![image](https://user-images.githubusercontent.com/113941541/191107444-3d3a66f7-3f68-4760-87e1-cbf7ec905781.png)

**Comment by iamromeo** [comment]
Hey @bluesidra  - Beautiful example - but this is not a bug

**Comment by bluesidra** [comment]
> Hey @bluesidra - Beautiful example - but this is not a bug

Hey @iamromeo! Thank you :) 
How do I get it resolved then? It's pretty annoying

**Comment by Kordishal** [comment]
@bluesidra I can't see any difference in behaviour on the two tabs?

When the text field has a scroll bar, the scroll is fixed to the text field and does not continue until you move the cursor out of the text field. Then it scrolls the page.

When the text field does not have a scroll bar it scrolls the page while the cursor is on it.

This behaviour is the same on both the content and section tabs for me (and anywhere on World Anvil).

**Comment by bluesidra** [comment]
@Kordishal 
Hi! It's different. I made a video, I hope it can explain it a bit better.
I also made that article public now: https://www.worldanvil.com/world/person/0864250f-e1a8-41c1-8758-29edd3a5d031/edit 
Hope that helps.
I've also tried different browsers now, and it's the same on both (Opera on a Ubuntu, Edge on a Win10)

https://user-images.githubusercontent.com/113941541/191243878-f6b0e3e0-64fc-46a5-9df3-6363fae5dc92.mp4

**Comment by Kordishal** [comment]
Hi, so that is a different article editor than I tested it with. The Plato / WYSIWYG does indeed continue to scroll sometime if it reaches the end of the window.

However, I am unable to reproduce your issue. Though I have only tried in Firefox so far. Either way if this is an issue with the editor it would not be addressed as that editor will be replaced sometime in the future.

Otherwise it might be something on your end that is causing this issue.

**Comment by bluesidra** [comment]
@Kordishal I am so sorry to bother you again, but I switched the editor to Euclid and it still doesn't work. I've now tried it on two different computers and three browsers. If it's on my end, it's to do with my WA account. 
And I checked again, it's not just over the text-editor, but over the entire div in which the text editors are in. It only appears if the cursor rest there for a while (>1sec). And the issue goes away when the page is on half size (like when I have it on one half of the screen and another app in the other half). I'm sorry to bother you, and I kinda thought more people had experienced it.
